### ISSUE TITLE
Remove _ in Suite and Test name in JUnit Reporter

### ISSUE THREAD

**Issue by AutomationReddy** [webdriverio/webdriverio#6400]
JUnit reporter outputs the test and suite name with an '_'. This actually looks odd or creating mismatching results with our Test management tool especially Zephyr

it's happening here:

```
private _prepareName (name = 'Skipped test') {
        return name.split(this._suiteNameRegEx).filter(
            (item) => item && item.length
        ).join('_')
    }
```
https://github.com/webdriverio/webdriverio/blob/afb20dee5c83e5d97bd90d0a95721f3c190502a3/packages/wdio-junit-reporter/src/index.ts#L36

Can we just leave the suite and test name as is?

**Comment by christian-bromann** [comment]
I am not sure, I think it was important for Jenkins as it doesn't allow any space.

**Comment by AutomationReddy** [comment]
@christian-bromann I believe it accepts names with spaces as well

**Comment by christian-bromann** [comment]
@AutomationReddy can you double check?

**Comment by AutomationReddy** [comment]
@christian-bromann Can you check this once? An example repo which parse XML to Jenkins Junit parser.

https://github.com/ATLANTBH/junit-results-parser/blob/master/examples/test.xml

**Comment by AutomationReddy** [comment]
@christian-bromann I have tested and this is working fine without _

**Comment by christian-bromann** [comment]
@AutomationReddy please provide a PR with the suggestions made here, thank you!

**Comment by AutomationReddy** [comment]
fixed in #6439
